src/Microsoft.AspNetCore.SystemWebAdapters.CoreServices/SessionState/Wrapped/AspNetCoreSessionManager.cs

```diff
@@ -23,4 +24,6 @@ public AspNetCoreSessionManager(ISessionKeySerializer serializer, ILoggerFactory
 
     public Task<ISessionState> CreateAsync(HttpContextCore context, SessionAttribute metadata)
         => Task.FromResult<ISessionState>(new AspNetCoreSessionState(context.Session, _serializer, _loggerFactory, metadata.IsReadOnly, _options.Value.ThrowOnUnknownSessionKey));
+
+    SessionStateMode ISessionManager.Mode => SessionStateMode.InProc;
```

Comment:
This is misleading, you can't tell if ISession is storing data in-proc or remote. Will that matter? 🤷

Reply:
Good call. After looking into it, the only thing that really makes sense for us to decalre ourselves as is Custom - everything else has a meaning.